### PR TITLE
Deploy instructions added to README

### LINKED ISSUE #121: Heroku deploy set up
## Description
Deploy to heroku https://dashboard.heroku.com/apps/contemplating-action/settings

## Plan
- [x] Create heroku account
- [x] Link to repo
- [x] Update readme - to deploy merge rc to 'master' and push to heroku 'master'
- [x] Add config / set up for heroku deploy
- [x] DNS for contemplating-action.org.uk @andybarnes  Please add hidden-longan-6yztcb9tvkqjdlebvgu5qki3.herokudns.com to the CNAME record for www.contemplatingaction.org.uk 

## Re-card for backlog
- [ ] Move staging to heroku?
- [ ] Set up to deploy auto deploy on tagged release?
- [ ] Add version number to app - at least package.json?

## Acceptance Criteria
- [ ] There is some information about heroku account and deploy instructions in the readme
- [ ] The site is live at https://www.contemplatingaction.org.uk

### PR DESCRIPTION
**Pull request by katjam**
Fixes #121

## Description

- Manual deploy process added to readme

@neontribe/contemplating-action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 # Picture Stories and Info
 
-## All the stuff
```
Comment (Nikomus) [inline README.md:3]: 👍

File: README.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+## Site urls
```
Comment (Nikomus) [inline README.md]:
```suggestion
## Site URLs
```

File: README.md [hunk 3]
```diff
@@ -32,5 +32,18 @@ We're using [elm-format](https://github.com/avh4/elm-format) to standardise form
 - Todo add to commit or push
 
 ## Deployment
+
+### To staging (github pages)
```
Comment (Nikomus) [inline README.md]:
```suggestion
### To staging (GitHub pages)
```

File: README.md [hunk 4]
```diff
@@ -32,5 +32,18 @@ We're using [elm-format](https://github.com/avh4/elm-format) to standardise form
...
 - Build to `dist\` with `yarn build`
 - Push `dist\*` to gh-pages branch
+
+### To Production (Heroku)
+- You will need a Heroku account with appropriate permissions
+- We're using the Heroku CLI(https://devcenter.heroku.com/articles/heroku-cli) for deploys. You will need to install it.
```
Comment (Nikomus) [inline README.md]:
```suggestion
- We're using the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) for deploys. You will need to install it.
```

**Review by Nikomus — DISMISSED** [review]
Some optional grammar pedantry in comments, but much better content 📜

**Comment by katjam** [comment]
@Nikomus accepted

**Review by Nikomus — APPROVED** [review]
Neat-o 🌻

**Pull request merged**